src/js/global-nav.js

```diff
@@ -286,9 +288,11 @@ function addListeners(breakpoint, wrapper) {
 
 export const createNav = ({ maxWidth = '68rem', showLogins = true } = {}) => {
   // Recruitment call to action
+  /* eslint-disable */
```

Comment:
If it's just for a lint it could be `// eslint-disable-next-line no-console` not to disable everything and no need to enable again later.

Reply:
```suggestion
// eslint-disable-next-line no-console
```

Reply:
I thought this was tripping the linter as Prettier will break this console.log over multiple lines but I just tested it and it seems to work ok - thanks!